src/lib/math.ts

```diff
@@ -70,8 +70,16 @@ export const loadMathJax = async (): Promise<boolean> => {
   }
 }
 
-export const mathifyElement = (elem: Element): void => {
+export const mathifyElement = async (elem: Element): Promise<void> => {
```

Comment:
Oh, I think you got the lint error because the merge made this async again. This line should be `export const mathifyElement = (elem: Element): void => {` and you can then back out the change to `ContentBlock.tsx`.